### PR TITLE
feat(post-process-forwarder): Move concurrency option to CLI

### PR DESCRIPTION
**Pull request by lynnagara**
This was an option previously as we were testing multiple values at runtime. Since this is no longer happening, move the concurrency setting to a CLI arg instead of a runtime option.

### CODE REVIEW

File: src/sentry/runner/commands/run.py [hunk 1]
```diff
@@ -335,6 +335,12 @@ def cron(**options):
     type=int,
     help="Time (in milliseconds) to wait before closing current batch and committing offsets.",
 )
+@click.option(
+    "--concurrency",
+    default=50,
```
Comment (lynnagara) [inline src/sentry/runner/commands/run.py:340]: @nikhars I believe this is the value that we determined to be optimal after testing. Is that accurate?

Comment (nikhars) [reply src/sentry/runner/commands/run.py:340]: 50 is too high. I think what we use in production is 10. What we should do is get this value from production and put that value as default.

At a higher concurrency level, the number of context switches would overwhelm the gain we get by having threads.

Comment (lynnagara) [reply src/sentry/runner/commands/run.py:340]: It definitely seemed high to me. But it's the number that appeared to be set in prod looking at our metrics.

Comment (lynnagara) [reply src/sentry/runner/commands/run.py]: Ok, the actual value we use is 5, I don't know why the metrics made it look like 50 to me. 5 seems like a more reasonable number to use across the board.

**Review by nikhars — APPROVED** [review]

**Pull request merged**